unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderEntity/DCLBuilderInWorldEntity.cs

```diff
@@ -360,21 +344,22 @@ private void DefaultAnimationStop()
 
     private void DefaultAnimationSample(float time)
     {
-        if (meshAnimations == null)
+        if (meshAnimations == null || meshAnimations.Length <= 0)
```

Comment:
As a `Length` (or `Count`) property cannot never be less than 0, I think the using of `== 0` will make the code more understandable than `<= 0`.

```suggestion
        if (meshAnimations == null || meshAnimations.Length == 0)
```